dotcom-rendering/src/web/components/App.tsx

```diff
@@ -117,15 +116,9 @@ export const App = ({ CAPI }: Props) => {
 		setBrazeMessages(buildBrazeMessages(CAPI.config.idApiUrl));
 	}, [CAPI.config.idApiUrl]);
 
-	const display: ArticleDisplay = decideDisplay(CAPI.format);
-	const design: ArticleDesign = decideDesign(CAPI.format);
 	const pillar: ArticleTheme = decideTheme(CAPI.format);
```

Comment:
We can remove this too, I guess?

Reply:
No, not yet. Some component is using it.